### DISCUSSION TITLE
Syntax error on all measures

### DISCUSSION THREAD

**Discussion by rjbartlejr** [DaxStudio/DaxStudio#480] [Q&A] [answered]
Every measure I write in Dax Studio it shows a syntax error - even if in Power BI the measure works perfectly. As an example, the measure:

POD Vacant = 
CALCULATE (
    [VacCnt],
    FILTER(qry_Employees,  [Division_Name] = "Plant Operations Division")
)

I get this error:

![image](https://user-images.githubusercontent.com/75798102/101793268-9a874980-3ad3-11eb-8509-e0d30e8db06e.png)

Am I doing something wrong or is this a bug?

**Comment by dgosbell** [comment, marked as answer]
This is not a bug. DAX Studio executes [DAX queries](https://docs.microsoft.com/en-us/dax/dax-queries) and the syntax you have there is not a valid dax query.

One possible query syntax for executing a measure like in your example and returning a single value would be something like the following:

DEFINE MEASURE 
qry_Employees[POD Vacant] =
CALCULATE (
[VacCnt],
FILTER(qry_Employees, [Division_Name] = "Plant Operations Division")
)
EVALUATE 
{ [POD Vacant] }
